tests/testthat/test-outpack-helpers.R

```diff
@@ -157,3 +157,33 @@ test_that("good error message if file not found in packet", {
       i = "For 'd.txt' did you mean 'a.txt', 'b.txt' or 'c.txt'",
       i = "Remember that all orderly paths are case sensitive"))
 })
+
+
+test_that("Can overwrite when copying files from packet", {
```

Comment:
ok, looks like this is due to

```
  outpack_packet_file_mark(packet, result$files$here, "immutable")
```

in the active context, so that's nice

Reply:
`file_mark` only adds a check the file's hash at the end of the packet run, it doesn't pro-actively prevent files from being modified (albeit the result is basically the same I suppose).

`file_export`, which I've modified here, does not get called by the `orderly_run` flow. I'll add a test still.

Reply:
Nope, I was wrong about that `file_export` not being called. With this PR, files in the draft folder become mutable again (as they were before #141).

The `outpack_packet_file_mark` still does it's job of checking at the end of the packet, so in my opinion that's fine. I've added an explicit test for this.